react/features/invite/components/add-people-dialog/web/InviteByEmailSection.tsx

```diff
@@ -98,7 +122,7 @@ function InviteByEmailSection({ inviteSubject, inviteText, inviteTextiOS }: IPro
             dispatch(showSuccessNotification({
                 titleKey: 'dialog.copied'
             }, NOTIFICATION_TIMEOUT_TYPE.SHORT));
-            copyText(inviteText);
+            _onCopyText();
```

Comment:
the success notification would be shown twice in this case, since _onCopyText also shows the notif.